api/db/seeds/seed.js

```diff
@@ -14,36 +15,102 @@ import { teamEvaluationDataBuilder } from './data/team-evaluation/data-builder.j
+
+  // Common
   await commonBuilder({ databaseBuilder });
   await tagsBuilder({ databaseBuilder });
 
   // FEATURES
+  // This is needed when you have to re-seed database that is fully migrated (ex: on Scalingo you can't drop database)
   await featuresBuilder({ databaseBuilder });
   await organizationLearnerImportFormat({ databaseBuilder });
 
   // ORGANIZATION
   await organizationBuilder({ databaseBuilder });
 
   // SCOPE
-  await teamPrescriptionDataBuilder({ databaseBuilder });
+  if (hasToSeed.prescription) {
+    logger.info('Seeding : Prescription');
+    await teamPrescriptionDataBuilder({ databaseBuilder });
+  }
 
-  await teamDevcompDataBuilder({ databaseBuilder });
+  if (hasToSeed.devcomp) {
+    logger.info('Seeding : Devcomp');
+    await teamDevcompDataBuilder({ databaseBuilder });
+  }
 
-  await teamAccesDataBuilder(databaseBuilder);
+  if (hasToSeed.acces) {
+    logger.info('Seeding : Acces');
+    await teamAccesDataBuilder(databaseBuilder);
+  }
 
-  await team1dDataBuilder(databaseBuilder);
+  if (hasToSeed.junior) {
+    logger.info('Seeding : Junior');
+    await team1dDataBuilder(databaseBuilder);
+  }
 
-  await teamContenuDataBuilder({ databaseBuilder });
+  if (hasToSeed.content) {
+    logger.info('Seeding : Contenu');
+    await teamContenuDataBuilder({ databaseBuilder });
+  }
 
-  await complementaryCertificationBuilder({ databaseBuilder });
-  await teamCertificationDataBuilder({ databaseBuilder });
+  if (hasToSeed.certification) {
+    logger.info('Seeding : Certification');
+    await complementaryCertificationBuilder({ databaseBuilder });
+    await teamCertificationDataBuilder({ databaseBuilder });
+  }
 
-  await teamEvaluationDataBuilder({ databaseBuilder });
+  if (hasToSeed.evaluation) {
+    logger.info('Seeding : evaluation');
+    await teamEvaluationDataBuilder({ databaseBuilder });
+  }
 
   await databaseBuilder.commit();
   await databaseBuilder.fixSequences();
+  logger.info('END Seeding');
 };
 
+function _buildContextToSeed(params) {
+  if (params.length === 0) {
+    return {
+      prescription: true,
+      devcomp: true,
+      acces: true,
+      junior: true,
+      content: true,
+      certification: true,
+      evaluation: true,
+    };
+  }
+
+  const context = {
+    prescription: false,
+    devcomp: false,
+    acces: false,
+    junior: false,
+    content: false,
+    certification: false,
+    evaluation: false,
+  };
+
+  params.forEach((seedContext) => {
+    if (seedContext === 'PRESCRIPTION') context.prescription = true;
+    if (seedContext === 'DEVCOMP') context.devcomp = true;
+    if (seedContext === 'JUNIOR') context.junior = true;
+    if (seedContext === 'ACCES') context.acces = true;
+    if (seedContext === 'CONTENT') context.content = true;
+    if (seedContext === 'CERTIF') context.certification = true;
+    if (seedContext === 'EVAL') context.evaluation = true;
+  });
```

Comment:
Juste pour savoir, pourquoi ne pas avoir fait un truc de ce genre ?
```suggestion
  const context = {
    prescription: seedContext === 'PRESCRIPTION',
    devcomp: seedContext === 'DEVCOMP',
    acces: seedContext === 'ACCES',
    junior: seedContext === 'JUNIOR',
    content: seedContext === 'CONTENT',
    certification: seedContext === 'CERTIF',
    evaluation: seedContext === 'EVAL',
  };
```

Reply:
je cloture cette question car j'avais mal lu le code 😓